### PR TITLE
create contributing file

### LINKED ISSUE #5: Create CONTRIBUTING.md
Refer  : https://github.com/OtacilioN/awesome-hacktoberfest-2018/blob/master/CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by pantharshit00**
closes #5

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Contribution Guidelines
+
+_Welcome hackers it is really **awesome** have you here! Before start contributing with this project make sure you read our [Code Of Conduct](https://github.com/0x48piraj/Awesome-Hacktoberfest/blob/master/CODE_OF_CONDUCT.md), it is really important to make this inclusive and open to everyone, otherwise, it would not be awesone_
+
+## Have you find some awesome Hacktoberfest participant repository?
```
Comment (0x48piraj) [inline CONTRIBUTING.md]: found*

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# Contribution Guidelines
+
+_Welcome hackers it is really **awesome** have you here! Before start contributing with this project make sure you read our [Code Of Conduct](https://github.com/0x48piraj/Awesome-Hacktoberfest/blob/master/CODE_OF_CONDUCT.md), it is really important to make this inclusive and open to everyone, otherwise, it would not be awesone_
```
Comment (0x48piraj) [inline CONTRIBUTING.md]: to have you here*

File: CONTRIBUTING.md (hunk 1 above)
Comment (0x48piraj) [inline CONTRIBUTING.md]: Add something like : "Add your achievements! What you did at this Hacktoberfest? Write it down, don't just do and forget."

**Comment by pantharshit00** [comment]
ok will add

**Comment by pantharshit00** [comment]
image


Just to get a link to include

**Comment by pantharshit00** [comment]
done! review please

**Review by 0x48piraj — APPROVED** [review]

**Comment by 0x48piraj** [comment]
Add your contribution too! Create a file. Get listed :)

**Pull request merged**